.gitignore

```diff
@@ -0,0 +1,7 @@
+# Compiled assets
```

Comment:
I recommend anchoring these paths to root for extra precision but we can do that another day.